esp/esp/program/modules/handlers/programprintables.py

```diff
@@ -788,22 +872,57 @@ def teacherschedules(self, request, tl, one, two, module, extra, prog):
 
         for teacher in teachers:
             # get list of valid classes
-            classes = [cls for cls in teacher.getTaughtSections()
-                    if cls.parent_program == self.program
-                    and cls.meeting_times.all().exists()
+            classes = [cls for cls in teacher.getTaughtSectionsFromProgram(self.program)
+                    if cls.meeting_times.all().exists()
                     and cls.resourceassignment_set.all().exists()
                     and cls.status > 0]
             # now we sort them by time/title
             classes.sort()
             for cls in classes:
                 scheditems.append({'name': teacher.name(),
                                    'teacher': teacher,
-                                   'cls' : cls})
+                                   'cls': cls})
 
         context['scheditems'] = scheditems
+        context['moderators'] = None
```

Comment:
Why `None` instead of `False`? (and in the next view as well)

Reply:
because to this day i havent figured out how to transmit False to django such that it doesnt convert it to a string that renders as true in the django if statement. help?